src/components/add/index.js

```diff
@@ -139,7 +139,10 @@ export default class Add extends React.Component {
   renderFooter = () => {
     return (
       <div css={style.footer}>
-        <FooterButton onClick={this.handleAddItemClick}>
+        <FooterButton
+          onClick={this.handleAddItemClick}
+          isApplying={this.state.isApplying}
+        >
           Add a First Item
```

Comment:
Make spinner an independent from buttons component so that it can be reused as a placeholder for loading pages and other elements as well. Use it here conditionally instead of the button text.

Reply:
I think, the spinner that Patryk suggested is good for the page, and not for the buttons. This spinner I wanted to keep for the footer button only. Initially, I did as a separate component and later realized that we probably will not use it as a separate component. Do you @rpavlovs think that we will use it as a separate component?

Reply:
I think we should use same spinner everywhere